src/sentry/dynamic_sampling/lastest_release_booster.py

```diff
@@ -0,0 +1,55 @@
+from datetime import datetime
+
+from django.core.cache import cache
```

Comment:
We should absolutely use Redis here. Initially, it should be fine to use the default cache, though we should consider to use the processing cluster after talking with ops.

```suggestion
from sentry.cache import default_cache
```